recipe/meta.yaml

```diff
@@ -134,6 +134,7 @@ outputs:
         - geotiff
         - giflib  # [not win]
         - json-c  # [not win]
+        - libgrpc
```

Comment:
We need to understand what dependency was bringing this in and either mark the previous build as broken or create a repodatapatch for this.

Reply:
I don't know the best approach when there's a dependency chain like this and the actual conda-forge pinning happens at the end of the chain, in this case downstream of tiledb and in google-cloud-cpp-feedstock repo, AFAIK

Reply:
Yep. It is super hard to figure out. I'm not blocking this PR and it should be merge but we need to dig a bit and ask around to avoid hindering a deeper problem or cluttering the gdal recipe with a fix that should be elsewhere.

TL;DR, let's add a comment on top of that line with the issue number, merge this, patch (preferred) or remove the previous build, and keep asking around until we figure this one out.

Reply:
In theory though, practically all the previous builds are under-specified. I want to review the build logs and then will bump this with your requested comment inclusion.

Reply:
The joys of maintaining gdal. I don't miss it ;-p